data/events/2022-lagos.yml

```
@@ -0,0 +1,94 @@
+name: "2022-lagos" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2022" # The year of the event. Make sure it is in quotes.
+city: "Lagos" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdayslag" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Lagos!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+cancel: "false"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: October 29th, 2022 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: October 29th, 2022 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: August 1st, 2022 # start accepting talk proposals.
+cfp_date_end: October 3rd, 2022 # close your call for proposals.
+cfp_date_announce: August 1st, 2022 # inform proposers of status
+
+cfp_link: "https://sessionize.com/localhost-conference-2022" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: August 1st, 2022 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: October 28th, 2022 # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "6.54451°, 3.39226°" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Lagos" # Defaults to city, but you can make it the venue name.
+#
+location_address: "Plot 9 Gbagada Industrial Scheme, Beside UPS, Gbagada Expressway" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
```

Keep this disabled until you're ready to accept sponsors (see also line 78).

```suggestion
 # - name: sponsor
```

---

We are actually ready to accept sponsorships, updated L78